tools/utils.py

```diff
@@ -85,7 +85,12 @@ def GuessArchitecture():
     return 'ia32'
   elif id == 'amd64':
     return 'ia32'
+  elif id.startswith('ppc'):
+    return 'ppc'
   else:
+    id = platform.processor()
+    if id == 'powerpc':
+      return 'ppc'
```

Comment:
When would this be reached?  Rephrased: is there ever a time when `platform.machine()` wouldn't return 'ppc' or 'ppc64'?